pom.xml

```diff
@@ -103,6 +103,10 @@
                 <scope>import</scope>
             </dependency>
             <dependency>
+                <groupId>software.amazon.awssdk</groupId>
+                <artifactId>bom</artifactId>
+                <version>${aws.awssdk.version}</version>
+                <type>pom</type>
                 <groupId>com.google.code.gson</groupId>
                 <artifactId>gson</artifactId>
             </dependency>
```

Comment:
@muuki88 you have an error in this `<dependency>` section which breaks build entirely.